### PR TITLE
Scc 4033/ds v3 update

### PR DESCRIPTION
**Pull request by charmingduchess**
Updates TagSet implementation to use new v3 DS update. Not sure why the vercel deployment is failing...

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
This is the error message in vercel:
Screen Shot 2024-03-22 at 2 52 18 PM

**Review by EdwinGuzman — COMMENTED** [review]
Just one nitpicky comment about tests but otherwise this works well.

File: src/components/RefineSearch/RefineSearch.tsx [hunk 1]
```diff
@@ -23,8 +23,10 @@ import type { Aggregation } from "../../types/filterTypes"
 
 interface RefineSearchProps {
   aggregations: Aggregation[]
-  setAppliedFilters: Dispatch<React.SetStateAction<Record<string, string[]>>>
-  appliedFilters: Record<string, string[]>
+  setAppliedFilters: Dispatch<
+    React.SetStateAction<CollapsedMultiValueAppliedFilters>
```
Comment (EdwinGuzman) [inline src/components/RefineSearch/RefineSearch.tsx:27]: Ah you need to import this.

**Review by EdwinGuzman — COMMENTED** [review]
I was trying to figure out if there's a way to catch the type issue before it's committed. You can try adding this to package.json: `"lint": "tsc --noEmit",` and then run `npm run lint`. This is the output:

Screen Shot 2024-03-22 at 3 11 44 PM

I think this check was removed when the husky script was updated?

**Comment by charmingduchess** [comment]
Yes the check was removed. That was the intention of updating the precommit hook. We probably should add it on a pre-push if we can.

**Review by EdwinGuzman — APPROVED** [review]

File: src/components/SearchFilters/appliedFilterUtils.test.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { buildAppliedFiltersValueArrayWithTagRemoved } from "./appliedFilterUtils"
+
+describe("Applied Filter utils", () => {
+  describe("buildAppliedFiltersValueArrayWithTagRemoved", () => {
```
[the remark this answers is not in the document]
Comment (charmingduchess) [reply src/components/SearchFilters/appliedFilterUtils.test.ts:7]: I added one test, I can't think of any other cases to test for.

**Pull request merged**